imports/zones/client.lua

```diff
@@ -328,13 +340,16 @@ lib.zones = {
         end
 
         data.coords = data.polygon:centroid()
+        data.type = 'poly'
         data.remove = removeZone
         data.contains = contains
+        data.setDebug = setDebug
 
         if data.debug then
             data.debug = nil
 
             CreateThread(function()
+                data.debugColour = vec4(data.debugColour?.r or 255, data.debugColour?.b or 42, data.debugColour?.g or 24, data.debugColour?.a or 100)
                 data.triangles = getTriangles(data.polygon)
                 data.debug = debugPoly
             end)
```

Comment:
Should probably just call `self:setDebug(...)` to cut down on duplicate code (here and on the other zone types).